docs/docs/spark-configuration.md

```diff
@@ -145,6 +145,59 @@ Using those SQL commands requires adding Iceberg extensions to your Spark enviro
 
 ## Runtime configuration
 
+### Precedence of Configuration Settings
+Iceberg allows configurations to be specified at different levels. The effective configuration for a read or write operation is determined based on the following order of precedence:
```

Comment:
This order doesn't look correct to me. Checking

https://github.com/apache/iceberg/blob/5ed88a5db0ffaea3fdd836c375fdce8d420d5956/spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkConfParser.java#L267-L300

The precedence is :

1. Option
2. Session
3. Table Property
4. Default

Reply:
Fixed, thanks.